app/models/event.rb

```diff
@@ -82,9 +82,18 @@ def self.format_date(type)
   end
 
   #scope
-  scope :recent_events, -> {order(created_at: :DESC).limit(12)}
-  scope :featured_events, -> {order(created_at: :DESC).limit(2)}
-  scope :popular_events, -> {order(created_at: :DESC).limit(3)}
+  scope :recent_events, -> { order(created_at: :DESC).limit(12) }
+  scope :featured_events, -> { order(created_at: :DESC).limit(12) }
+  scope :popular_events, -> lambda do
```

Comment:
Unused block argument - `lambda`. If it's necessary, use `_` or `_lambda` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.<br>Use the `lambda` method for multi-line lambdas.